x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -55,3 +55,5 @@ export const installationStatuses = {
   Installed: 'installed',
   NotInstalled: 'not_installed',
 } as const;
+
+export const learnMoreBlogPost = 'https://ela.st/learn-agent-integrations';
```

Comment:
I guess this constant is not used and we prefer to use the doc link service right?